common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -93,68 +93,40 @@ public static String modifyMultipleMatchKeyword(String query){
                         Expression<Long> conceptIdCheck = root.get("conceptId");
                         synonymConceptPredicate.add(conceptIdCheck.in(synonymConceptIds));
-                        predicates.add(criteriaBuilder.or(standardConceptPredicates.toArray(new Predicate[0])));
-                        if (standardConceptFilter.equals(StandardConceptFilter.STANDARD_CONCEPTS) || standardConceptFilter.equals(StandardConceptFilter.STANDARD_OR_CODE_ID_MATCH)) {
-                                predicates.add(criteriaBuilder.or(synonymConceptPredicate.toArray(new Predicate[0])));
-                        } else if (standardConceptFilter.equals(StandardConceptFilter.NON_STANDARD_CONCEPTS)) {
-                            predicates.add(criteriaBuilder.or(
-                                            criteriaBuilder.or(criteriaBuilder.isNull(root.get("standardConcept"))),
-                                            criteriaBuilder.and(nonStandardConceptPredicates.toArray(new Predicate[0]))));
-                        }
+                        predicates.add(criteriaBuilder.or(synonymConceptPredicate.toArray(new Predicate[0])));
                     }
-                    else{
-                        List<Predicate> conceptCodeIDName = new ArrayList<>();
-                        Expression<Double> matchExp = null;
-                        if(!Strings.isNullOrEmpty(query)){
-                            final String keyword = modifyMultipleMatchKeyword(query);
-                            conceptCodeIDName.add(criteriaBuilder.equal(root.get("conceptCode"),
+                    else {
+                        List<Predicate> conceptCodeID = new ArrayList<>();
+                        if (!Strings.isNullOrEmpty(query)) {
+                            // adds check ?(conceptCode == query) o the predicate list
+                            conceptCodeID.add(criteriaBuilder.equal(root.get("conceptCode"),
                                     criteriaBuilder.literal(query)));
                             try {
+                                // adds check ?(conceptId == query) to the predicate list
                                 long conceptId = Long.parseLong(query);
-                                conceptCodeIDName.add(criteriaBuilder.equal(root.get("conceptId"),
+                                conceptCodeID.add(criteriaBuilder.equal(root.get("conceptId"),
                                         criteriaBuilder.literal(conceptId)));
                             } catch (NumberFormatException e) {
                                 // Not a long, don't try to match it to a concept ID.
                             }
-                            matchExp = criteriaBuilder.function("match", Double.class, root.get("conceptName"), criteriaBuilder.literal(keyword));
+
+                            predicates.add(criteriaBuilder.or(conceptCodeID.toArray(new Predicate[0])));
                         }
 
+
+                    }
+
```

Comment:
Can you clean up indention and and add some line breaks to the really long lines in this sub and file?